pydm/data_plugins/__init__.py

```diff
@@ -87,10 +119,34 @@ def load_plugins_from_path(locations, token):
     return added_plugins
 
 
+def is_read_only():
+    """
+    Check whether or not the app is running with the read only flag set.
+
+    Returns
+    -------
+    bool
+        True if read only. False otherwise.
+    """
+    return __read_only
+
+
+def set_read_only(read_only):
```

Comment:
I would make sure these get bubbled up to top level imports. Might also be nice to put an info logging message so that we don't confuse ourselves trying to figure out why everything is read-only

Reply:
Done with the latest commits.